webclient/http1/src/main/java/io/helidon/webclient/http1/Http1ClientResponseImpl.java

```diff
@@ -96,7 +97,7 @@ class Http1ClientResponseImpl implements Http1ClientResponse {
         this.whenComplete = whenComplete;
 
         if (responseHeaders.contains(HeaderNames.CONTENT_LENGTH)) {
```

Comment:
there is a method `contentLength()` on server response headers that can be used

Reply:
thx!